backend/package.json

```diff
@@ -22,6 +22,13 @@
     "@types/passport": "^1.0.11",
     "@types/passport-google-oauth20": "^2.0.11",
     "@types/passport-local": "^1.0.35",
+    "@typescript-eslint/eslint-plugin": "^5.52.0",
+    "eslint": "^8.34.0",
+    "eslint-config-standard-with-typescript": "^34.0.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^15.6.1",
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-plugin-react": "^7.32.2",
```

Comment:
do you know if we need these packages here? looks like we installed these at the root level already?

Reply:
I don't think we need these packages here. These earlier commits were back when I was trying to get the linter running inside the backend folder, but now that we've decided to run the linter in root, all of these changes can be discarded/deleted.

Reply:
let's remove then!